### ISSUE TITLE
Containers are stuck in "Networking" state in 2 of the hosts. 

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#2058]
Server version - v0.37.1

When I was executing some LB related test cases , I got in to a situation where LB agent gets stuck in "Networking" state . After all containers that get deployed in this host is stuck in "networking" state forever. This was first seen in the Digital Ocean machine .

After a while , I see the same issue happen in another custom host where I see LB agent stuck in "Networking" state . After all containers that get deployed in this host is stuck in "networking" state forever. 

Following is the exception I see repeatedly in the server logs:

```
2015-09-17 02:58:27,374 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Requesting update of item(s) [healthcheck, host-iptables, ipsec-hosts, hosts, iptables] on [agent:6] 
2015-09-17 02:58:27,375 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:6] on [healthcheck], not in sync requested [72] != applied [29] 
2015-09-17 02:58:27,375 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:6] on [host-iptables], not in sync requested [63] != applied [24] 
2015-09-17 02:58:27,376 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:6] on [ipsec-hosts], not in sync requested [63] != applied [24] 
2015-09-17 02:58:27,376 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:6] on [hosts], not in sync requested [82] != applied [26] 
2015-09-17 02:58:27,376 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:6] on [iptables], not in sync requested [65] != applied [26] 
2015-09-17 02:58:27,377 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Requesting update of item(s) [healthcheck, host-iptables, ipsec-hosts, hosts, iptables] on [agent:8] 
2015-09-17 02:58:27,377 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:8] on [healthcheck], not in sync requested [72] != applied [27] 
2015-09-17 02:58:27,378 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:8] on [host-iptables], not in sync requested [63] != applied [18] 
2015-09-17 02:58:27,378 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:8] on [ipsec-hosts], not in sync requested [63] != applied [20] 
2015-09-17 02:58:27,378 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:8] on [hosts], not in sync requested [82] != applied [22] 
2015-09-17 02:58:27,378 INFO  [:] [] [] [] [ecutorService-7] [.p.c.v.i.ConfigItemStatusManagerImpl] Waiting on [agent:8] on [iptables], not in sync requested [65] != applied [22] 
2015-09-17 02:58:33,532 DEBUG [:] [] [] [] [ecutorService-4] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:230] on resource [22] 
2015-09-17 02:58:33,532 DEBUG [:] [] [] [] [ecutorService-4] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:230] on resource [22] 
2015-09-17 02:58:33,537 DEBUG [:] [] [] [] [cutorService-34] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:239] on resource [23] 
2015-09-17 02:58:33,538 DEBUG [:] [] [] [] [cutorService-34] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:239] on resource [23] 
2015-09-17 02:58:33,542 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [service.activate:277] on resource [13] 
2015-09-17 02:58:33,542 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [service.activate:277] on resource [13] 
2015-09-17 02:58:33,544 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [service.activate:279] on resource [15] 
2015-09-17 02:58:33,544 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [service.activate:279] on resource [15] 
2015-09-17 02:58:33,550 DEBUG [:] [] [] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:284] on resource [32] 
2015-09-17 02:58:33,550 DEBUG [:] [] [] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:284] on resource [32] 
2015-09-17 02:58:33,554 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:294] on resource [33] 
2015-09-17 02:58:33,554 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:294] on resource [33] 
2015-09-17 02:58:33,558 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:295] on resource [34] 
2015-09-17 02:58:33,559 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:295] on resource [34] 
2015-09-17 02:58:33,569 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:326] on resource [38] 
2015-09-17 02:58:33,571 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:326] on resource [38] 
2015-09-17 02:58:33,575 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:327] on resource [39] 
2015-09-17 02:58:33,575 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:327] on resource [39] 
2015-09-17 02:58:33,580 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:329] on resource [40] 
2015-09-17 02:58:33,580 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:329] on resource [40] 
2015-09-17 02:58:33,585 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [service.activate:276] on resource [12] 
2015-09-17 02:58:33,585 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [service.activate:276] on resource [12] 
2015-09-17 02:58:33,590 DEBUG [:] [] [] [] [cutorService-26] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [service.activate:366] on resource [21] 
2015-09-17 02:58:33,595 DEBUG [:] [] [] [] [cutorService-28] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:371] on resource [48] 
2015-09-17 02:58:33,595 DEBUG [:] [] [] [] [cutorService-28] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:371] on resource [48] 
2015-09-17 02:58:33,597 DEBUG [13320c7a-40ed-4cec-83dd-b9ef54fcdfb3:366] [service:21] [service.activate->(LoadBalancerServiceActivatePreListener)] [] [cutorService-26] [c.p.e.p.i.DefaultProcessInstanceImpl] Running pre listener [LoadBalancerServiceActivatePreListener] 
2015-09-17 02:58:33,597 DEBUG [13320c7a-40ed-4cec-83dd-b9ef54fcdfb3:366] [service:21] [service.activate->(LoadBalancerServiceActivatePreListener)] [] [cutorService-26] [c.p.e.p.i.DefaultProcessInstanceImpl] Finished pre listener [LoadBalancerServiceActivatePreListener] 
2015-09-17 02:58:33,601 DEBUG [:] [] [] [] [cutorService-39] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:352] on resource [46] 
2015-09-17 02:58:33,601 DEBUG [:] [] [] [] [cutorService-39] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:352] on resource [46] 
2015-09-17 02:58:33,605 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:401] on resource [55] 
2015-09-17 02:58:33,614 DEBUG [:] [] [] [] [cutorService-27] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:306] on resource [37] 
2015-09-17 02:58:33,614 DEBUG [:] [] [] [] [cutorService-27] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:306] on resource [37] 
2015-09-17 02:58:33,616 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:378] on resource [51] 
2015-09-17 02:58:33,619 DEBUG [:] [] [] [] [cutorService-32] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [TIMEOUT] process [service.activate:277] on resource [13] 
2015-09-17 02:58:33,620 INFO  [:] [] [] [] [cutorService-32] [i.c.p.e.e.i.ProcessEventListenerImpl] Communication timeout on process [service.activate:277] on [13] : Object [instance:34] failed to satisfy predicate [15000] millis 
2015-09-17 02:58:33,620 DEBUG [:] [] [] [] [cutorService-33] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [TIMEOUT] process [service.activate:276] on resource [12] 
2015-09-17 02:58:33,621 INFO  [:] [] [] [] [cutorService-33] [i.c.p.e.e.i.ProcessEventListenerImpl] Communication timeout on process [service.activate:276] on [12] : Object [instance:32] failed to satisfy predicate [15000] millis 
2015-09-17 02:58:33,623 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:341] on resource [42] 
2015-09-17 02:58:33,623 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:341] on resource [42] 
2015-09-17 02:58:33,624 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [service.activate:414] on resource [29] 
2015-09-17 02:58:33,624 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [service.activate:414] on resource [29] 
2015-09-17 02:58:33,627 DEBUG [:] [] [] [] [cutorService-14] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [TIMEOUT] process [loadbalancerhostmap.create:328] on resource [4] 
2015-09-17 02:58:33,628 INFO  [:] [] [] [] [cutorService-14] [i.c.p.e.e.i.ProcessEventListenerImpl] Communication timeout on process [loadbalancerhostmap.create:328] on [4] : Object [instance:46] failed to satisfy predicate [15000] millis 
2015-09-17 02:58:33,628 DEBUG [:] [] [] [] [cutorService-38] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:378] on resource [51] 
2015-09-17 02:58:33,629 DEBUG [:] [] [] [] [cutorService-24] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [TIMEOUT] process [service.activate:414] on resource [29] 
2015-09-17 02:58:33,629 INFO  [:] [] [] [] [cutorService-24] [i.c.p.e.e.i.ProcessEventListenerImpl] Communication timeout on process [service.activate:414] on [29] : Object [instance:62] failed to satisfy predicate [15000] millis 
2015-09-17 02:58:33,630 DEBUG [13320c7a-40ed-4cec-83dd-b9ef54fcdfb3:366] [service:21] [service.activate->(ServiceUpdateActivate)] [] [cutorService-26] [c.p.e.p.i.DefaultProcessInstanceImpl] Running handler [ServiceUpdateActivate] 
2015-09-17 02:58:33,631 DEBUG [:] [] [] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [loadbalancerhostmap.create:328] on resource [4] 
2015-09-17 02:58:33,632 DEBUG [:] [] [] [] [cutorService-28] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:377] on resource [50] 
2015-09-17 02:58:33,634 DEBUG [:] [] [] [] [ecutorService-1] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:401] on resource [55] 
2015-09-17 02:58:33,635 DEBUG [:] [] [] [] [cutorService-31] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [TIMEOUT] process [service.activate:279] on resource [15] 
2015-09-17 02:58:33,635 INFO  [:] [] [] [] [cutorService-31] [i.c.p.e.e.i.ProcessEventListenerImpl] Communication timeout on process [service.activate:279] on [15] : Object [loadBalancerHostMap:3] failed to satisfy predicate [15000] millis 
2015-09-17 02:58:33,636 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:392] on resource [54] 
2015-09-17 02:58:33,636 DEBUG [:] [] [] [] [cutorService-30] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:392] on resource [54] 
2015-09-17 02:58:33,637 DEBUG [:] [] [] [] [cutorService-27] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:428] on resource [62] 
2015-09-17 02:58:33,637 DEBUG [:] [] [] [] [cutorService-27] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:428] on resource [62] 
2015-09-17 02:58:33,638 DEBUG [:] [] [] [] [ecutorService-4] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:430] on resource [63] 
2015-09-17 02:58:33,638 DEBUG [:] [] [] [] [ecutorService-4] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:430] on resource [63] 
2015-09-17 02:58:33,640 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:532] on resource [65] 
2015-09-17 02:58:33,642 DEBUG [e0f88506-b9b8-44bd-a55b-22624627e32a:328] [LoadBalancerHostMap:4] [loadbalancerhostmap.create->(ActivateByDefault)] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Running post listener [ActivateByDefault] 
2015-09-17 02:58:33,643 DEBUG [:] [] [] [] [cutorService-39] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:418] on resource [58] 
2015-09-17 02:58:33,643 DEBUG [:] [] [] [] [cutorService-34] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:424] on resource [61] 
2015-09-17 02:58:33,643 DEBUG [e0f88506-b9b8-44bd-a55b-22624627e32a:328] [LoadBalancerHostMap:4] [loadbalancerhostmap.create->(ActivateByDefault)] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Finished post listener [ActivateByDefault] 
2015-09-17 02:58:33,644 DEBUG [:] [] [] [] [cutorService-39] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:418] on resource [58] 
2015-09-17 02:58:33,644 DEBUG [:] [] [] [] [cutorService-28] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:377] on resource [50] 
2015-09-17 02:58:33,645 DEBUG [e0f88506-b9b8-44bd-a55b-22624627e32a:328] [LoadBalancerHostMap:4] [loadbalancerhostmap.create->(LoadBalancerUpdateConfig)] [] [cutorService-35] [c.p.e.p.i.DefaultProcessInstanceImpl] Running post listener [LoadBalancerUpdateConfig] 
2015-09-17 02:58:33,645 DEBUG [:] [] [] [] [cutorService-34] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:424] on resource [61] 
2015-09-17 02:58:33,646 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Attempting to run process [instance.create:454] on resource [64] 
2015-09-17 02:58:33,647 DEBUG [:] [] [] [] [cutorService-37] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:454] on resource [64] 
2015-09-17 02:58:33,647 DEBUG [:] [] [] [] [cutorService-36] [c.p.e.p.i.DefaultProcessInstanceImpl] Exiting [PROCESS_ALREADY_IN_PROGRESS] process [instance.create:532] on resource [65] 
```

**Comment by ibuildthecloud** [comment]
https://github.com/rancher/cattle/pull/885

**Comment by sangeethah** [comment]
Tested with server  version - v0.38.0-rc2

This issue was not encountered when  executing similar set of  LB related test cases . Closing this issue. Will open a new issue , if I happen to encounter similar situation.
